stdlib/distutils/dist.pyi

```diff
@@ -7,7 +7,7 @@ from typing import IO, Any
 command_re: Pattern[str]
 
 class DistributionMetadata:
-    def __init__(self, path: FileDescriptorOrPath | None = None) -> None: ...
+    def __init__(self, path: FileDescriptorOrPath | None = ...) -> None: ...
```

Comment:
Why have you removed the default value here, and from several other functions in this PR? We've been trying to add default values where possible over the last few months, as they're very helpful for IDE users. It's also helped us expose a number of latent bugs in typeshed that previously went unnoticed (e.g. see #9608).

Reply:
Oops, I thought it was preferable to avoid coping the default, and this was due to copying. I'll revert that.

Reply:
We changed our policy fairly recently on this, no worries :)

Reply:
In my defense, these were inconsistent between the two files, so I was just making this one match the other one.

Reply:
Yes, well, we haven't finished deciding exactly how many defaults we want added to the stdlib yet (https://github.com/PyCQA/flake8-pyi/issues/357) — give us time to move on to the third-party stubs!!